packages/test-runner-commands/test/send-mouse/sendMousePlugin.test.ts

```diff
@@ -0,0 +1,79 @@
+import path from 'path';
+import { platform } from 'os';
+import selenium from 'selenium-standalone';
+import { runTests } from '@web/test-runner-core/test-helpers';
+import { chromeLauncher } from '@web/test-runner-chrome';
+import { webdriverLauncher } from '@web/test-runner-webdriver';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+import { sendMousePlugin } from '../../src/sendMousePlugin';
+import { startSeleniumServer } from '../selenium-server';
+
+let seleniumServer: selenium.ChildProcess;
+
+describe('sendMousePlugin', function test() {
+  this.timeout(20000);
+
+  before(async function () {
+    seleniumServer = await startSeleniumServer({
+      chrome: { version: '94.0.4606.41' },
+      firefox: { version: 'latest' },
+    });
+  });
+
+  after(() => {
+    seleniumServer.kill();
+  });
+
+  it('can send mouse on puppeteer', async () => {
+    await runTests({
+      files: [path.join(__dirname, 'browser-test.js')],
+      browsers: [chromeLauncher()],
+      plugins: [sendMousePlugin()],
+    });
+  });
+
+  // playwright doesn't work on windows VM right now
+  if (platform() !== 'win32') {
```

Comment:
You can remove this check, most of Playwright tests have been passing lately.

Reply:
Done.